README.md

```diff
@@ -65,58 +50,146 @@ To clean up files generated from the build, you can simply run:
+- `-f` or `--function` - Function to Invoke
+- `-d` or `--data` - Stringified JSON data to use as either query params or request body
+- `-p` or `--path` - Path to JSON file to use as either query params or request body
+- `-m` or `--method` - HTTP method for request
 
-If at any point, you no longer need your service, you can run the following command to remove the Azure Functions that were created, and ensure you don't incur any unexpected charges:
+##### Example
+
+After deploying template function app, run
 
 ```bash
-serverless remove
-``` 
+$ sls invoke -f hello '{"name": "Azure"}'
+```
 
-### Advanced Authentication
+### Roll Back Your Function App
+
+To roll back your function app to a previous deployment, simply select a timestamp of a previous deployment and use `rollback` command.
+
+```bash
+# List all deployments to know the timestamp for rollback
+$ sls deploy list
+Serverless:
+-----------
+Name: myFunctionApp-t1561479533
+Timestamp: 1561479533
+Datetime: 2019-06-25T16:18:53+00:00
+-----------
+Name: myFunctionApp-t1561479506
+Timestamp: 1561479506
+Datetime: 2019-06-25T16:18:26+00:00
+-----------
+Name: myFunctionApp-t1561479444
+Timestamp: 1561479444
+Datetime: 2019-06-25T16:17:24+00:00
+-----------
+
+# Rollback Function App to timestamp
+$ sls rollback -t 1561479506
+```
+
+This will update the app code and infrastructure to the selected previous deployment.
+
+For more details, check out our [rollback docs](docs/ROLLBACK.md).
+
+### Deleting Your Function App
 
-The getting started walkthrough illustrates the interactive login experience, which is recommended for most users. However, if you'd prefer to create an Azure ["service principal"](https://github.com/Azure/azure-sdk-for-node/blob/master/Documentation/Authentication.md#2-azure-cli) yourself, you can indicate that this plugin should use its credentials instead, by setting the following environment variables:
+If at any point you no longer need your service, you can run the following command to delete the resource group containing your Azure Function App and other depoloyed resources using:
 
-**Bash**
 ```bash
-export azureSubId='<subscriptionId>'
-export azureServicePrincipalTenantId='<tenantId>'
-export azureServicePrincipalClientId='<servicePrincipalName>'
-export azureServicePrincipalPassword='<password>'
+$ sls remove
 ```
 
-**Powershell**
-```powershell
-$env:azureSubId='<subscriptionId>'
-$env:azureServicePrincipalTenantId='<tenantId>'
-$env:azureServicePrincipalClientId='<servicePrincipalName>'
-$env:azureServicePrincipalPassword='<password>'
+### Creating or removing Azure Functions
+
+To create a new Azure Function within your function app, run the following command from within your app's directory:
+
+```bash
+# -n or --name for name of new function
+$ sls func add -n {functionName}
```

Comment:
this doesn't work either for me - command func not found

Reply:
@ahemavathy This makes me think you might be working off of an outdated version. Are you using npm link? Where did you install the plugin from?